assets/dev/js/editor/controls/slider.js

```diff
@@ -213,6 +220,20 @@ ControlSliderItemView = ControlBaseUnitsItemView.extend( {
 
 		this.$el.remove();
 	},
+
+	onDeviceModeChange() {
+		const currentDeviceMode = elementor.channels.deviceMode.request( 'currentMode' ),
+			isMobile = 'mobile' === currentDeviceMode,
+			isMobileValue = this.model.get( 'name' ).includes( '_mobile' ),
+			hasDefault = this.model.get( 'default' ),
+			shouldRunConversion = isMobile && isMobileValue && hasDefault && this.isCustomUnit();
+
+		if ( shouldRunConversion ) {
+			setTimeout( () => {
```

Comment:
Defer to wait for render to complete (before `maybeDoFractionToCustomConversions()` might trigger a re-render).